src/main/java/frc/robot/subsystems/HubVision.java

```diff
@@ -49,7 +46,7 @@
          * @param rotationSpeed Rotation speed
          */
         public arcadeDriveSpeeds(double fowardSpeed, double rotationSpeed) {
-            this.maxSpeed = 0.5;
+            this.maxSpeed = 1;
```

Comment:
Please move this to Constants

Reply:
That should not be in constants, the theory is that anyone defining an arcadeDriveSpeeds needs to willingly bypass the half motor speed clamp. If you really want the value removed, however, it would be better to remove the constructor entirely and force the developer to give it a max speed.